example/android/android-studio-robolectric-example/.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: android
```

Comment:
Yeah, or we could add this into coshx/docker-bdd-example as an example of a working cluster with a rails server and an android client.



> On Feb 12, 2015, at 1:10 AM, Gabe Kopley <notifications@github.com> wrote:
> 
> In example/android/android-studio-robolectric-example/.travis.yml:
> 
> > @@ -0,0 +1,10 @@
> > +language: android
> Honestly ideally these example projects would be submodules.
> 
> —
> Reply to this email directly or view it on GitHub.
>